aten/src/ATen/native/native_functions.yaml

```diff
@@ -8322,14 +8322,15 @@
     CompositeExplicitAutograd: elu_
 
 - func: glu.out(Tensor self, int dim=-1, *, Tensor(a!) out) -> Tensor(a!)
+  structured: True
   python_module: nn
   dispatch:
     CPU, CUDA: glu_out
 
 - func: glu(Tensor self, int dim=-1) -> Tensor
+  structured_delegate: glu.out
+  device_check: NoCheck   # TensorIterator
```

Comment:
not sure if this is correct. reading the docs on structured native functions (in particular `device_check`) it seems like `TensorIterator::borrowing_binary_op` should be performing this check already.